reana/cli.py

```diff
@@ -1905,38 +1946,77 @@ def run_example(
 @click.option(
     "-m",
     "--mount",
+    "mounts",
+    multiple=True,
+    callback=volume_mounts_to_list,
+    help="Which local directories whould be mounted in the Kind nodes? local_path=reana_system_path",
```

Comment:
Can you propagate the same `-m` new behaviour to the `run_ci()` command as well?  It would be practical for CI runs.

Reply:
(so that `-m` behaves the same for all commands it exists for...)

Reply:
Added. Note that due to https://github.com/helm/helm/issues/5618 we have to make the implementation a bit more convoluted. However, the final result is more streamlined with the way we do the `cluster-create`, where we print the config that is being applied, in this case the modified `values.yaml` content, so it turns out to be good in my opinion. 
<details><summary>Example output</summary>
<p>

```
$ reana-dev cluster-deploy -j /code:/var/code                                                                                                      [11:17:18] 
[2020-06-25T11:17:18] reana: cat <<EOF | helm install reana helm/reana -n default --create-namespace --wait -f -                                                                                                                      
components:                                                                                                        
  reana_job_controller:                                                                                                                                                                                                               
    image: reanahub/reana-job-controller                                                                           
  reana_message_broker:                                                                                                                                                                                                               
    image: reanahub/reana-message-broker                 
  reana_server:                                                                                                                                                                                                                       
    image: reanahub/reana-server                                                                                   
  reana_ui:                                                                                                        
    image: reanahub/reana-ui                                                                                       
  reana_workflow_controller:                                                                                       
    environment:                                                                                                   
      REANA_JOB_HOSTPATH_MOUNTS: '[{"name": "var-code", "hostPath": "/code", "mountPath":                          
        "/var/code"}]'                                                                                             
    image: reanahub/reana-workflow-controller                                                                      
  reana_workflow_engine_cwl:                                                                                       
    image: reanahub/reana-workflow-engine-cwl                                                                      
  reana_workflow_engine_serial:                                                                                                                                                                                                       
    image: reanahub/reana-workflow-engine-serial                                                                                                                                                                                      
  reana_workflow_engine_yadage:                                                                                    
    image: reanahub/reana-workflow-engine-yadage                                                                   
notifications:                                                                                                                                                                                                                        
  email_config:                                                                                                    
    receiver: team@reana.io                                                                                        
    sender: notifications@reana.io                                                                                                                                                                                                    
  enabled: true                                                                                                                                                                                                                       
  system_status: '*/5 * * * *'    

NAME: reana                                                                                                        
LAST DEPLOYED: Thu Jun 25 11:17:19 2020                                                                                                                                                                                               
NAMESPACE: default                                                                                                                                                                                                                    
STATUS: deployed                                                                                                   
REVISION: 1                                                                                                                                                                                                                           
TEST SUITE: None                                                                                                                                                                                                                      
NOTES:                                                                                                             
The REANA system has been installed:                                                                               
                                                                                                                   
If you are installing REANA for the first time, there are a few steps left to finalise its configuration.          
                                                                                                                   
1. Get the REANA-Server pod name:                                                                                  
                                                                                                                   
    $ REANA_SERVER=$(kubectl get pod -l "app=reana-server" -o name -o jsonpath='{.items[0].metadata.name}')                                                                                                                           
                                                                                                                   
2. Initialise the database:                                                                                        
                                                                                                                   
    $ kubectl exec $REANA_SERVER -- ./scripts/setup                                                                                                                                                                                   
                                                                                                                                                                                                                                      
3. Create your administrator user and store the token:                                                                                                                                                                                

    $ kubectl exec $REANA_SERVER -- flask reana-admin create-admin-user john.doe@example.org
    <reana-admin-access-token-value>
    $ read -s REANA_ADMIN_ACCESS_TOKEN  # paste the secret here
    $ kubectl create secret generic reana-admin-access-token \
      --from-literal=ADMIN_ACCESS_TOKEN="$REANA_ADMIN_ACCESS_TOKEN"

Thanks for flying REANA 🚀
[2020-06-25T11:17:33] reana: kubectl config set-context --current --namespace=default
Context "kind-kind" modified.
[2020-06-25T11:17:33] reana: /Users/rodrigdi/code/reanahub/reana/scripts/wait-for-db.sh
reana-db-f5494cb59-rbg77
Waiting for REANA-DB to be ready.
Waiting for REANA-DB to be ready.
Waiting for REANA-DB to be ready.
REANA-DB ready
[2020-06-25T11:17:49] reana: /Users/rodrigdi/code/reanahub/reana/scripts/create-admin-user.sh
/Users/rodrigdi/code/reanahub/reana/scripts/create-admin-user.sh: line 18: not: command not found
Defaulting container name to rest-api.
Use 'kubectl describe pod/reana-server-85bd5d6448-x6mjm -n default' to see all of the containers in this pod.
Creating REANA database...
DB Created.
REANA database created.
Creating Invenio database...
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

Creating database postgresql+psycopg2://reana:reana@reana-db.default.svc.cluster.local:5432/reana
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '

Creating all tables!
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Created all tables!
Invenio database created.
Defaulting container name to rest-api.
Use 'kubectl describe pod/reana-server-85bd5d6448-x6mjm -n default' to see all of the containers in this pod.
/usr/local/lib/python3.6/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to su
ppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
secret/reana-admin-access-token created                                                                                                                                                                                                    
```


</p>
</details>